app/helpers/datasets_helper.rb

```diff
@@ -85,6 +85,11 @@ def metadata_files(dataset)
     files
   end
 
+  def licence_url_exits?(dataset)
+    dataset.licence_url.nil? ? "#{request.protocol}#{request.host_with_port}#{request.fullpath}#license-info" : dataset.licence_url
```

Comment:
```suggestion
    dataset.licence_url.presence || "#{request.protocol}#{request.host_with_port}#{request.fullpath}#license-info"
```

This is [a nice little method provided by Rails](https://apidock.com/rails/Object/presence) to avoid needing to reference the `dataset.licence_url` variable twice.

Reply:
This line is too long to read comfortably, I think it would be better as a full `if`.

Reply:
Ah nice thanks @thomasleese ! I've updated this now 👍